oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/web/authentication/OAuth2ErrorAuthenticationFailureHandler.java

```diff
@@ -47,14 +49,18 @@ public final class OAuth2ErrorAuthenticationFailureHandler implements Authentica
 	private final Log logger = LogFactory.getLog(getClass());
 	private HttpMessageConverter<OAuth2Error> errorResponseConverter = new OAuth2ErrorHttpMessageConverter();
 
+	private BiConsumer<OAuth2AuthenticationException, ServletServerHttpResponse> httpResponseCustomizer = (exception, httpResponse) -> httpResponse.setStatusCode(HttpStatus.BAD_REQUEST);
+
+	private Function<OAuth2AuthenticationException, OAuth2Error> errorCustomizer = OAuth2AuthenticationException::getError;
```

Comment:
I don't think this is needed as I can customize the error parameters by providing a custom `Converter` to `OAuth2ErrorHttpMessageConverter.setErrorParametersConverter()`.

For example, I can customize `OAuth2ClientAuthenticationFilter.authenticationFailureHandler` to use `OAuth2ErrorAuthenticationFailureHandler` but with a custom `OAuth2ErrorHttpMessageConverter.setErrorParametersConverter()` that only returns the `OAuth2Error.errorCode`.